js/background.js

```diff
@@ -17,7 +17,7 @@ chrome.runtime.onMessageExternal.addListener(function(request, sender, sendRespo
 });
 
 chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
-  if (tab.url && tab.url.indexOf('https://www.facebook.com') === 0) {
+  if (tab.url && /https?:\/\/(www|web)\.facebook\.com.*/.test(tab.url)) {
```

Comment:
@x0rm3r Can you update it? Also please cache the regex before using, so we don't have to re-create it on every `onUpdate` calls.